pom.xml

```diff
@@ -0,0 +1,109 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.kooperate</groupId>
+  <artifactId>conifg-model-project</artifactId>
+  <packaging>pom</packaging>
+  <version>0.1-SNAPSHOT</version>
+  <name>Kooperate :: POM</name>
+
+  <properties>
+    <format.skip>false</format.skip>
+    <java.source>1.8</java.source>
+    <java.target>1.8</java.target>
```

Comment:
As this project is new, I think we should start using at least Java 11.

Reply:
It seems that jkube is also using 1.8 so ....